### DISCUSSION TITLE
Use property no longer supports github repo url

### DISCUSSION THREAD

**Discussion by tavurth** [vercel/vercel#4091] [General] [answered]
From the [DEVELOPING A RUNTIME](https://github.com/zeit/now/blob/master/DEVELOPING_A_RUNTIME.md) docs: 

> the use property will work with any npm install argument such as a git repo url which is useful for testing your Runtime.

But when attempting to use a git repo url it always breaks with error about invalid semver.

<img width="1021" alt="Screenshot 2020-04-17 at 12 18 13" src="https://user-images.githubusercontent.com/100964/79539309-d0d80a80-80a5-11ea-86b2-d954e54c77f8.png">

I would consider this a minor regression, and [it seems to have been introduced 6 months ago](https://github.com/zeit/now/blame/7ca672a470a1cb4114ac0864658ce64ed81accd0/packages/now-build-utils/src/detect-builders.ts#L548)

**Comment by styfle** [comment, marked as answer]
The document you linked to is describing the [`builds`](https://zeit.co/docs/configuration#project/builds) property.

The [`functions`](https://zeit.co/docs/configuration#project/functions) property does not accept all the same options.

**Comment by tavurth** [comment]
Thank you!
